### PR TITLE
Add StakeRun endpoints & style 

### PR DESCRIPTION
**Pull request by Errorist79**
- Added StakeRun (g)RPC/WS/LCD/Archival endpoints
- separated the gRPC and gRPC-Web columns

### CODE REVIEW

File: data/resources.json [hunk 1]
```diff
@@ -57,6 +57,10 @@
         {
           "title": "Swagger",
           "value": "https://lcd-axelar.imperator.co/static/swagger/"
+        },
+        {
+          "title": "Swagger",
+          "value": "https://lcd-axelar.stakerun.com/static/swagger/"
```
Comment (StephenFluin) [inline data/resources.json:63]: Can you change the titles so that it's clear the difference?

File: data/resources.json [hunk 2]
```diff
@@ -322,12 +352,17 @@
...
+    {
+      "name": "gRPC-Web endpoints",
+      "value": [
         {
-          "value": "axelartest-grpcweb.quantnode.tech:9091"
+          "value": "https://axelartest-grpcweb.quantnode.tech"
         },
         {
-          "value": "https://axelartest-grpcweb.quantnode.tech"
+          "value": "https://axelartest-grpcweb.quantnode.tech:9091"
```
Comment (StephenFluin) [inline data/resources.json:365]: Is this duplication correct?

**Review by StephenFluin — CHANGES_REQUESTED** [review]
Requested some small changes and a double check. Thanks for this contribution!

File: data/resources.json (hunk 2 above)
Comment (Errorist79) [reply data/resources.json:365]: i did not remove the fields in the previous document. (`..grpcweb.quantnode.tech:9091` and `grpcweb.quantnode.tech`.) but i think we can rm this one (at L365)

File: data/resources.json (hunk 1 above)
Comment (Errorist79) [reply data/resources.json:63]: sure

**Review by milapsheth — APPROVED** [review]

**Pull request merged**